charts/team-ns/templates/networkpolicy.yaml

```diff
@@ -2,7 +2,21 @@
 {{- $v := .Values | merge (dict) }}
 {{- $prometheus := dig "managedMonitoring" "prometheus" false $v }}
 {{- $alertmng := dig "managedMonitoring" "alertmanager" false $v }}
-{{- if and (not (eq $v.teamId "admin")) (dig "networkPolicy" "ingressPrivate" true $v) }}
+{{- if (not (dig "networkPolicy" "ingressPrivate" true $v)) }}
+---
+# If team network policies are disable then we whitelist all traffic to prvent undesired blocking while deploying team workloads
```

Comment:
"If team network policies are disable then we whitelist all traffic to prvent undesired blocking while deploying team workloads" -> "If team network policies are disabled then we whitelist all traffic to prevent undesired blocking while deploying team workloads"